### ISSUE TITLE
[Question] How do I run this against multiple routes?

### ISSUE THREAD

**Issue by CodyBontecou** [netlify/netlify-plugin-lighthouse#458]
I'm curious how i can run lighthouse against multiple paths within my web app? I've looked around and attempted a few of the examples but have had no luck. This is my current netlify.toml

```
[[plugins]]
  package = "@netlify/plugin-lighthouse"

  [plugins.inputs.thresholds]
    performance = 0.9
    accessibility = 0.9
    best-practices = 0.9
    seo = 0.9

  [plugins.inputs]
    output_path = "reports/lighthouse.html"
  
    [plugins.inputs.audits]
      path = "how-to-use-vuetify-with-nuxt-3"
```

**Comment by kevinsamoei** [comment]
Here is an example:

if your site's publish directory is `dist`, assume you have the following structure:
```
.
├── README.md
├── netlify.toml
├── dist
│   ├── about
│   │   └── index.html
│   ├── blog
│   │   └── index.html
│   └── index.html
├── package-lock.json
└── package.json

```

you would then have this in your `netlify.toml`

```
[plugins.inputs]
output_path = "reports/lighthouse.html"

[[plugins.inputs.audits]]
output_path = "reports/blog-report.html"
path = "blog"

[[plugins.inputs.audits]]
path = "about"

[[plugins.inputs.audits]]
path = ""
```

Let us know if this does it

**Comment by CodyBontecou** [comment]
Thank you, @kevinsamoei, for taking a look. Unfortunately, I'm not having any luck.

This is the directory structure of my dist folder:
<img width="400" alt="Screen Shot 2022-08-23 at 8 47 27 AM" src="https://user-images.githubusercontent.com/20440899/186239282-87cb6766-fe50-4099-bb6b-362431245669.png">

Here's my `netlify.toml`:
```
[[plugins.inputs.audits]]
  output_path = "reports/lighthouse.html"
  
[[plugins.inputs.audits]]
  output_path = "reports/how-to-use-vuetify-with-nuxt-3-report.html"
  path = "how-to-use-vuetify-with-nuxt-3.html"

[[plugins.inputs.audits]]
  path = ""
```

This is what I'm seeing within my Netlify dashboard:
<img width="781" alt="Screen Shot 2022-08-23 at 8 51 28 AM" src="https://user-images.githubusercontent.com/20440899/186240031-46da65fd-55fb-4eb7-a08b-bfded105f52c.png">

**Comment by aitchiss** [comment]
Hi @CodyBontecou 👋 

This feature is now available in `v4.0.0` (implemented in https://github.com/netlify/netlify-plugin-lighthouse/pull/487) so please do upgrade! You will now be able to specify, e.g.

```
[[plugins.inputs.audits]]
  path = "how-to-use-vuetify-with-nuxt-3.html"
```

Please do let us know if you have any issues, and thanks so much for reaching out with this question!

**Comment by CodyBontecou** [comment]
Nice! Thank you @aitchiss 

It took me a little while to get to this, but it looks like it's working perfectly.
